x-pack/solutions/observability/packages/kbn-genai-cli/README.md

```diff
@@ -0,0 +1,15 @@
+# @kbn/genai-cli
+
+Repository of GenAI scripts, both for practical and educational purposes.
+
+Usage:
+
+```ts
+import { runRecipe } from '../utils/run_recipe';
+
+runRecipe(async ({ inferenceClient, kibanaClient, log, signal }) => {
+  const output = await inferenceClient.output({});
+
+  log.info(output);
+});
```

Comment:
This example seems pretty short. Intentional?

Reply:
yes - there are more verbose examples in kbn-inference-cli